packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -777,7 +777,7 @@ async function createApp(
   let workspaceSetupSpinner = ora('Creating your workspace').start();
 
   try {
-    const fullCommand = `${pmc.exec} nx ${command} --nxWorkspaceRoot=${nxWorkspaceRoot}`;
+    const fullCommand = `${pmc.exec} nx ${command} --nxWorkspaceRoot="${nxWorkspaceRoot}"`;
```

Comment:
I don't think this PR makes sense anymore. The lines 763...776 already implement wrapping in double quotes by default + escaping quotes for Windows and old npm.

The following code would result in e.g. `npx nx new .... --nxWorkspaceRoot=""path/to/my/root""` or in case of Windows and npm 6 in `npx nx new .... --nxWorkspaceRoot="\\"path/to/my/root\\""`

Reply:
https://github.com/nrwl/nx/blob/master/packages/create-nx-workspace/bin/create-nx-workspace.ts#L808

Yes, think so...